src/listpack.c

```diff
@@ -3185,6 +3209,55 @@ int listpackTest(int argc, char *argv[], int flags) {
         printf("Done. usec=%lld\n\n", usec()-start);
     }
 
+    TEST("Test lpFindInteger") {
+        // Create a listpack with a mix of integers and strings
```

Comment:
```suggestion
        /* Create a listpack with a mix of integers and strings. */
```
same as other places.